uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Component/GroupButtonItem.php

```diff
@@ -100,10 +100,8 @@ public function addBottomMargin() {
    */
   public function render() {
     $buttonMarkup = '
-      <a href="%s">
-        <button class="%s">
-          <span><i class="%s"></i></span> %s
-        </button>
+      <a href="%s" class="%s">
```

Comment:
shouldn't a button always have the `btn` class, at least, by default? Where do the classes come from? Do we have to specify `btn` every time?

Reply:
Right! makes sense to  add `btn` by default as currently we have to add it every time. The class are defined in file https://github.com/civicrm/civihr/blob/b0bbf29773846f64cb6ca211352699686022e0a1/uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Helper/UserInformationActionGroup.php#L100